src/inspect.js

```diff
@@ -2,15 +2,17 @@ import fs from 'fs';
 import path from 'path';
 import semver from 'semver';
 import logger from '@percy/logger';
-import migrations from './migrations';
-import { run } from './utils';
+import { run, ROOT, migration } from './utils.js';
+import { getPackageJSON } from '@percy/cli-command/utils';
 
 // Tries to detect the installed SDK by checking the current project's CWD. Checks non-dev deps in
 // addition to dev deps even though SDKs should only be installed as dev deps.
-function inspectPackageJSON(info) {
+async function inspectPackageJSON(info) {
   try {
-    let pkg = require(`${process.cwd()}/package.json`);
+    let pkg = getPackageJSON(process.cwd());
+    if (!pkg) throw Error('Missing package.json');
```

Comment:
Seems unnecessary to throw this error when all we're doing is swallowing it and logging. Could move the log here and return early. Can also make use of `log.warn` returning `undefined` (which is what this function does on error).

```suggestion
    if (!pkg) return log.warn('Could not find package.json in current directory');
```